tests/system/action/user/test_create.py

```diff
@@ -290,68 +346,236 @@ def test_create_permission_superadmin(self) -> None:
+                "structure_level_$": ["1"],
+                "structure_level_$1": "structure_level 1",
+                "vote_weight_$": ["1"],
+                "vote_weight_$1": "12.002345",
+                "about_me_$": ["1"],
+                "about_me_$1": "about me 1",
+                "comment_$": ["1"],
+                "comment_$1": "comment zu meeting/1",
+                "vote_delegations_$_from_ids": ["1"],
+                "vote_delegations_$1_from_ids": [5, 6],
+                "meeting_ids": [1],
             },
         )
 
-    def test_create_permission_committee_manager_no_permission(self) -> None:
-        """ vote_weight_$ is in group B and may only work with meeting-Permission  """
+    def test_create_permission_group_B_user_can_manage_no_permission(self) -> None:
+        """ Group B fields needs explicit user.can_manage permission for meeting """
+        self.permission_setup()
+        self.set_organisation_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+        self.set_user_groups(self.user_id, [3])  # Empty group of meeting/1
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "usersname",
+                "number_$": {"1": "number1"},
+                "group_$_ids": {"1": [1]},
+            },
+        )
+        self.assert_status_code(response, 403)
+        self.assertIn(
+            "You are not allowed to perform action user.create. Missing permission: Permission user.can_manage in meeting 1",
+            response.json["message"],
+        )
+
+    def test_create_permission_group_C_oml_manager(self) -> None:
+        """ May create group C group_$_ids by OML permission """
+        self.permission_setup()
+        self.set_organisation_management_level(
+            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        )
+
+        response = self.request(
+            "user.create",
+            {
+                "username": "usersname",
+                "group_$_ids": {1: [1]},
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "user/3",
+            {"group_$_ids": ["1"], "group_$1_ids": [1], "username": "usersname"},
+        )
+
+    def test_create_permission_group_C_committee_manager(self) -> None:
+        """ May create group C group_$_ids by committee permission """
         self.permission_setup()
         self.update_model(
             f"user/{self.user_id}",
             {
-                "committee_ids": [60],
-                "organisation_management_level": OrganisationManagementLevel.CAN_MANAGE_USERS,
-                "group_$_ids": ["1", "4"],
-                "group_$1_ids": [2],  # admin group of meeting/1
-                "group_$4_ids": [4],  # default group of meeting/4
+                "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
```

Comment:
Here again: The user is not in the `committee/user_ids`, but has a CML set. IMO this must be catched.

Reply:
Is catched now explicitly in update.py respectively create.py. But IMO would be better to remove committee_ids field